components/brave_wallet/browser/json_rpc_service.cc

```diff
@@ -1577,7 +1614,8 @@ void JsonRpcService::EnsGetEthAddr(const std::string& domain,
       allow_offchain = false;
     }
 
-    // JsonRpcService owns EnsResolverTask instance, so Unretained is safe here.
+    // JsonRpcService owns EnsResolverTask instance, so Unretained is safe
+    // here.
```

Comment:
Undo comment changes here and above

Reply:
https://github.com/brave/brave-core/pull/18768/commits/9f425b93043a567489ab276ac78daffefc4ce1f5